ups/validate_drp.table

```diff
@@ -4,6 +4,7 @@ setupRequired(afw)
 setupRequired(pipe_base)
 setupRequired(pyyaml)
 
+setupOptional(pipe_tasks)
 setupOptional(obs_cfht)
 setupOptional(obs_decam)
 setupOptional(validation_data_cfht)
```

Comment:
I wanted to ask about these lines (the obs_ packages and validation data package). The ReadMe says to set them up, but the table file already does it. I am a bit uncomfortable with the redundancy. If any of these are needed for unit tests, bin/ scripts or library code then clearly they have to stay (but then they should not be optional, either). If they aren't needed for the above then I recommend removing these 4 lines and leaving the ReadMe as it stands.

It's a messy business trying to get this right. I hope validate_drp will work with any camera, in which case favoring these two is a bit odd. This is especially true because these cameras are not yet part of lsst_distrib, making them much harder for a user to install and keep properly updated -- a fact that has burned me several times, including yesterday. I think importing pipe_tasks and not obs_* is probably a good compromise, but I'm not sure.